src/vstart.sh

```diff
@@ -595,7 +595,7 @@ prepare_conf() {
         heartbeat file = $CEPH_OUT_DIR/\$name.heartbeat
 "
 
-    local mgr_modules="iostat"
+    local mgr_modules="iostat nfs"
```

Comment:
No need to enable it here.  We already enable it when `NFS=1`.
https://github.com/ceph/ceph/blob/546b653c45a0c64f3e09b532b604b3ac6da350f2/src/vstart.sh#L1461

Reply:
i think it's better to enable it by default (as we do now in master for real clusters) so that you can 'nfs cluster create' when cephadm is enabled.

Reply:
I'll remove that other enable line